transport-classes-io_uring/src/main/java/io/netty/incubator/channel/uring/AbstractIOUringStreamChannel.java

```diff
@@ -211,12 +211,13 @@ protected Executor prepareToClose() {
 
         private ByteBuf readBuffer;
         private IovArray iovArray;
+        private final ByteBuf iovArrayBuffer = alloc().directBuffer(Limits.IOV_MAX * IovArray.IOV_SIZE);
 
         @Override
         protected int scheduleWriteMultiple(ChannelOutboundBuffer in) {
             assert iovArray == null;
             int numElements = Math.min(in.size(), Limits.IOV_MAX);
-            ByteBuf iovArrayBuffer = alloc().directBuffer(numElements * IovArray.IOV_SIZE);
+            iovArrayBuffer.capacity(numElements * IovArray.IOV_SIZE);
             iovArray = new IovArray(iovArrayBuffer);
```

Comment:
@mind-fuzzy  I would check if we can reuse `IovArray` somehow, which is a minor but annoying allocation in the hot path